tests/test_wave_01.py

```diff
@@ -11,7 +11,7 @@ def test_create_movie_all_params_valid_returns_movie():
 
     # Act
     new_movie = create_movie(movie_title, genre, rating)
-
+    print(new_movie)
```

Comment:
It’s usually best to remove "debugging" print statements like this before committing and/or submitting.